packages/core/test/unit/utils/HtmlValidationUtil.test.ts

```diff
@@ -0,0 +1,20 @@
+import { readFileSync } from 'fs';
+import * as path from 'path';
+import * as logger from '../../../src/utils/logger';
+import { checkForVueHydrationWithRules } from '../../../src/utils/htmlValidationUtil';
+
+const table_without_tbody = readFileSync(path.resolve(__dirname, 'html_str/table_without_tbody.txt'), 'utf8');
+const correct_html_string = readFileSync(path.resolve(__dirname, 'html_str/correct_html.txt'), 'utf8');
+
+describe('checkForVueHydrationWithRules', () => {
+  it('should not log an error when the content is correct', () => {
```

Comment:
```suggestion
  it(' should not log an error when the table has tbody tag', () => {
```

Add white space so that the test name would be 
'checkForVueHydrationWithRules should not log an error when the table has tbody'

Reply:
Oh I see, thank you for pointing this nit.